### DISCUSSION TITLE
Раздражающее смещение диалогов

### DISCUSSION THREAD

**Discussion by omqmo** [taiga-family/taiga-ui#5060] [Q&A] [answered]
Диалоги дергуются в сторону при открытии и закрытии, что изрядно раздражает. Дергается на десктопе. Это как-то можно исправить?
Примеры из документации - 
(дергается при открытии) https://taiga-ui.dev/components/dialog#header (фулскрин версия)
(дергается при закрытии) https://taiga-ui.dev/components/dialog#mobile

UPD: Понял из-за чего так. Подобное происходит по причине установки стилей -webkit-scrollbar, хотя и не понятно зачем это, так как сам скрол бар скрыт. Наверняка есть возможность его отображать и вот было бы вообще круто, если бы и стили устанавливались только при таком сценарии.  Мне кажется это логичным.

**Comment by waterplea** [comment, marked as answer]
Это баг, вроде раньше не было. Слетела кастомизация скролла, наверное. Посмотрим.

UPDATE: В актуальной версии исправлено.
